bucket/grpcurl.json

```diff
@@ -0,0 +1,34 @@
+{
+    "homepage": "https://github.com/fullstorydev/grpcurl",
+    "description": "Command-line tool for interacting with gRPC servers",
+    "license": "MIT",
+    "version": "1.5.0",
```

Comment:
```suggestion
    "version": "1.6.0",
    "description": "cURL like utility for gRPC",
    "homepage": "https://github.com/fullstorydev/grpcurl",
    "license": "MIT",
```

Reply:
Description, version number, (including hashes and artifact URLs), and property order have been updated to match suggested change.